draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -1012,47 +1036,58 @@ terminated and the associated keying material discarded.
-
-   The initial DTLS connection will be used to establish a new endpoint pair shared
-   secret which MUST use shared key identifier 1. The endpoint pair shared
-   secret are derived using the TLS exporter interface using the ASCII string
-   "EXPORTER-DTLS-OVER-SCTP-EXT" with no terminating NUL, no context, and length
-   64.
+   empty and MUST be used by both endpoints when establishing the first DTLS connection.
+
+   The initial DTLS connection will be used to establish two new endpoint pair shared
+   secrets which MUST use shared key identifier 2 and 3. The endpoint pair shared
+   secrets are derived using the TLS exporter interface using the ASCII strings
+   "EXPORTER-DTLS-OVER-SCTP-CLIENT-WRITE" and "EXPORTER-DTLS-OVER-SCTP-SERVER-WRITE"
+   with no terminating NUL, no context, and length 64. Keys derived with the label
+   "EXPORTER-DTLS-OVER-SCTP-CLIENT-WRITE" are used by the TLS client for sending SCTP-AUTH
+   and always have an odd Shared Key Identifier. Keys derived with the label
+   "EXPORTER-DTLS-OVER-SCTP-SERVER-WRITE" are used by the TLS server for sending SCTP-AUTH
+   and always have an even Shared Key Identifier.
 
    ~~~~~~~~~~~
-   TLS-Exporter("EXPORTER-DTLS-OVER-SCTP-EXT", , 64)
+   TLS-Exporter("EXPORTER-DTLS-OVER-SCTP-CLIENT-WRITE", , 64)
+   TLS-Exporter("EXPORTER-DTLS-OVER-SCTP-SERVER-WRITE", , 64)
    ~~~~~~~~~~~
 
-   When a subsequent DTLS connection is setup, a new 64-byte endpoint pair shared
-   secret is derived using the TLS-Exporter as defined above. The Shared Key
-   Identifiers form a sequence. If the previous endpoint pair shared secret used
-   Shared Key Identifier n, the new one MUST use Shared Key Identifier
-   n+1, unless n = 65535, in which case the new Shared Key Identifier
-   is 1.
+   When a subsequent DTLS connection is setup, two new 64-byte endpoint pair shared
+   secrets are derived using the TLS-Exporter as defined above. The Shared Key
+   Identifiers form a sequence. If the previous endpoint pair shared secrets used
+   Shared Key Identifiers 2n and 2n+1, the new ones MUST use Shared Key Identifier
+   2n+2 and 2n+3, unless 2n = 65534, in which case the new Shared Key Identifiers
+   are 2 and 3.
 
 ### DTLS 1.2 Considerations
 
-   Whenever a new DTLS connection is established, a 64-byte endpoint pair shared secret
-   is derived using the TLS-Exporter described in {{RFC5705}}.
+   Whenever a new DTLS connection is established, two 64-byte endpoint pair shared secrets
+   are derived using the TLS-Exporter described in {{RFC5705}}.
 
    After sending the DTLS Finished message
    for the initial DTLS connection, the active SCTP-AUTH key MUST be
-   switched from key identifier 0 to key identifier 1. Once the
+   switched from key identifier 0 to key identifiers 2 and 3. Once the
    initial Finished message from the peer has been processed by DTLS,
    the SCTP-AUTH Shared Key Identifier zero MUST NOT be used.
```

Comment:
Should the "zero (0)" be used to make it clear when one talk about key identifier 0 above?

Reply:
fixed